src/siembol/resources/logger.py

```diff
@@ -0,0 +1,60 @@
+"""
```

Comment:
The question where is good place for this script for testing?

Reply:
Do we still want this, even?  I thought we were going to remove this in favor of a different approach to generating sample log messages.

Reply:
yes we need to think about it to do it simple. Meybe something for testing can be in https://github.com/G-Research/siembol-config

Reply:
ok, but we can remove it from this repo, for sure.  I don't think it's being used by anything anymore.  Correct me if I'm wrong and we should put it back.